tests/e2e/commands/fillInput.js

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
one small nit: can you please change the file year to 2017 here and in the other files as well :)